amlip_cpp/src/cpp/node/collaborative_learning/ModelManagerSenderNode.cpp

```diff
@@ -68,6 +68,17 @@ ModelManagerSenderNode::~ModelManagerSenderNode()
     logDebug(AMLIPCPP_DDS_MODELMANAGERSENDER, "ModelManagerSender Node Destroyed.");
 }
 
+void ModelManagerSenderNode::update_statistics(
+        const std::string& name,
+        const std::string& data)
+{
+    statistics_.name(name);
+    statistics_.data(data);
+
+    // Send statistics
+    statistics_writer_->publish(statistics_);
+}
```

Comment:
Reuse code, you could call the other method from within this one.

Reply:
Good, but I think the copy can be avoided.

Reply:
Same in `modelManagerTest.cpp` and `model_manager_sender.cpp`.
Or better, use here the new method.